GVFS/GVFS.FunctionalTests/FileSystemRunners/CmdRunner.cs

```diff
@@ -93,6 +93,14 @@ public override string ReplaceFile(string sourcePath, string targetPath)
             return this.RunProcess(string.Format("/C move /Y \"{0}\" \"{1}\"", sourcePath, targetPath));
         }
 
+        public override void ReplaceFile_AccessShouldBeDenied(string sourcePath, string targetPath)
+        {
+            // CMD does not report any error messages when access is denied, so just confirm the file still exists
+            this.ReplaceFile(sourcePath, targetPath);
+            this.FileExists(sourcePath).ShouldEqual(true);
+            this.FileExists(targetPath).ShouldEqual(false);

```

Comment:
Nit: These can be `ShouldBeTrue` and `ShouldBeFalse`.  Could you also include messages like you did in the other runners?

Reply:
Sure!  I was just emulating what was done in each runner's `DeleteFile_AccessShouldBeDenied()` method.  What I've done is just align the `CmdRunner` pair with all the others now.